middlewares/ProfilesMiddleware.js

```diff
@@ -0,0 +1,39 @@
+import Response from '../helpers/statusResponse';
+
+// This function checks for user id being an integer on users profile creation
+const checkUsersId = (req, res, next) => {
+  const {
+    userId
+  } = req.params;
+
+  if (!Number.isInteger(+userId)) {
+    res.status(400).json({
+      errors: {
+        body: [
+          'User Id must be an integer'
+        ]
+      }
+    });
+  }
+  return next();
+};
+
+// This fucntion checks for a user entering valid inputs users profile update
+const validProfileInput = (req, res, next) => {
+  if (req.body.username === '' || req.body.biodata === '' || req.body.image === '' || req.body.address === '' || req.body.dateofbirth === '') {
```

Comment:
Use a more generic falsy validation. Example `!req.body.biodata` would catch for all falsy value. What you have is only validating for empty strings, anything other than this, it will not catch it.